packages/omg-js-util/src/transaction.js

```diff
@@ -342,6 +342,9 @@ const transaction = {
   *
   */
   encodeMetadata: function (str) {
+    if (str.startsWith('0x')) {
```

Comment:
we can use validateMetadata here

Reply:
i added the validateMetadata logic to the joi validation instead, i think this encode should just take care of encoding and not do validation